todo/src/scripts/storageController.js

```diff
@@ -0,0 +1,88 @@
+class StorageController {
+    constructor(name) {
+        this.init();
+        this.data;
+    }
+
+    init() {
+        if (!localStorage.getItem('toDo')){
+            localStorage.setItem('toDo', JSON.stringify(
+                {
+                    'currentTasks': [],
+                    'completeTasks': [],
+                }
+            ));
+        }
+        this.data = JSON.parse(localStorage.getItem('toDo'));
+        this.removeEmptyItems();
+    }
+
+    removeEmptyItems() {
+        this.data['currentTasks'] = this.data['currentTasks'].filter((item) => {return item != null});
```

Comment:
why do you need to use brackets for fields ?

Reply:
I remove this method, but deleted all brackets in similar cases.
```suggestion
sortTasksId() {
        this.data.currentTasksId = this.data.currentTasksId.sort((a, b)=> a - b);
        this.data.completeTasksId = this.data.completeTasksId.sort((a, b)=> a - b);  
    }
```